dags/atd_knack_arterial_managment_locations.py

```diff
@@ -17,7 +17,7 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "retries": 0,
-    "retry_delay": duration(minutes=5),
+    "execution_timeout": duration(minutes=30),
```

Comment:
same as other comment, does this take a while to run?

Reply:
nope again—thank you!